app/decorators/container_manager_decorator.rb

```diff
@@ -0,0 +1,5 @@
+class ManageIQ::Providers::ContainerManager < MiqDecorator
```

Comment:
10x, I actually needed to remove this one, I added it before I found we already have an existing
```app/decorators/manageiq/providers/openshift/container_manager_decorator.rb```